core/src/main/java/com/linecorp/armeria/client/retry/RetryingHttpClient.java

```diff
@@ -77,7 +77,7 @@
      */
     public static Function<Client<HttpRequest, HttpResponse>, RetryingHttpClient>
     newDecorator(RetryStrategy retryStrategy, int maxTotalAttempts) {
-        return new RetryingHttpClientBuilder(retryStrategy)
+        return RetryingHttpClient.builder(retryStrategy)
                 .maxTotalAttempts(maxTotalAttempts)
                 .newDecorator();
```

Comment:
How about aligning at dot (`.`)?

    return RetryingHttpClient.builder(retryStrategy)
                             .maxTotalAttempts(...)
                             ...

Please fix all other similar issues in this PR as well.

Reply:
Yes, That's more looks neat and clear. I'll change all similar code to the suggested style.